docs/mkdocs/docs/api/macros/index.md

```diff
@@ -29,6 +29,7 @@ header. See also the [macro overview page](../../features/macros.md).
 
 ## Type conversions
 
+- [**JSON_DISABLE_ENUM_SERIALIZATION**](json_disable_enum_serialization.md) - switch off default serialization/deserialization functions for enums
```

Comment:
Seems like this belongs right next to `NLOHMANN_JSON_SERIALIZE_ENUM`

Reply:
I still believe it belongs under `Serialization/deserialization macros` as I've mentioned before.

Reply:
I don't have any further wish here. Good to have #3540 for this.